python/ccxt/bybit.py

```diff
@@ -1684,14 +1749,22 @@ def cancel_order(self, id, symbol=None, params={}):
                 request['order_id'] = id
                 method = 'privatePostV2PrivateOrderCancel'
         elif self.is_spot():
-            request['orderId'] = id
+            order_link_id = self.safe_string(params, 'order_link_id')
+            if order_link_id:
+                request['orderLinkId'] = order_link_id
+                params = self.omit(params, "order_link_id")
+            else:
+                request['orderId'] = id
             if is_conditional:
```

Comment:
are we sending the stop_order_id also in ByBit SPOT? cause if not it will always be orderCategory=0

Reply:
thinking about this, wouldn't it be better to send "type" in here as well and be consistent with:
is_conditional = order_type == 'stop'